### ISSUE TITLE
Suport for IBM Codeengine as executer for notebooks/python scirpts/piplines

### ISSUE THREAD

**Issue by romeokienzler** [elyra-ai/elyra#937]
IBM Code Engine will soon provide a "Supercomputer as a Service" where tasks can be scaled on 1000s of cores for just a couple of seconds. Would be interesting to look at how an integration can be done at different levels:
- notebook cell
- notebook
- python script
- python function
- pipeline

**Comment by ptitzler** [comment]
IBM Cloud Code Engine is currently in beta. https://cloud.ibm.com/docs/codeengine?topic=codeengine-getting-started

**Comment by romeokienzler** [comment]
yes - so Elyra support also should be either beta as well or we wait until Code Engine is production ready, but then we will loose time for adoption. I think it would be a very nice feature to just push computation elastically to CodeEngine...
